common/src/main/java/dev/isxander/yacl3/gui/controllers/ColorController.java

```diff
@@ -210,11 +228,63 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
         @Override
         public boolean mouseClicked(double mouseX, double mouseY, int button) {
             if (super.mouseClicked(mouseX, mouseY, button)) {
+                //Controller support/keyboard only support(enter key to "select" hue/saturation&value areas?)
+                //Detects if the user has clicked the color preview
+                if(isMouseOverColorPreview(mouseX, mouseY)) {
+                        playDownSound();
+                        createOrRemoveColorPicker();
+                }
```

Comment:
The controller/keyboard support comment was originally an idea I set out to do, but decided against because I didn't know how. Thought I removed that comment but I didn't.

The rest of the code is responsible for handling the color picker creation and deletion if a user clicked on the color preview.